Update some examples to use proper YAML syntax.

jcsmith opened the pull request:
##### SUMMARY
Many of the examples in the ansible documentation use the name=value syntax instead of the proper YAML name: value syntax.  As I understand it there is now a preference for using the name: value syntax.  I have updated the examples in docs/docsite/rst/playbooks_blocks.rst to use this syntax and verified they all pass ansible-playbook --syntax-check when they are included in a playbook.

If this PR is well received I will work on updating the other examples.  I just wanted to get this published to gather feedback before updating other examples. 

 - Docs Pull Request

##### COMPONENT NAME
playbooks_blocks.rst

Commit:
Update some examples to use proper YAML syntax.

mkrizek commented:
cc @gundalow

gundalow commented on docs/docsite/rst/playbooks_blocks.rst:
```diff
@@ -93,3 +93,17 @@ Another example is how to run handlers after an error occurred :
 
 
 
+  tasks:
```
Looks like this got pasted in the wrong place. Should be line 81,

jcsmith commented:
Thanks for the feed back. I’ll update and push again later today. Have to get a few things done for $DAYJOB first.

Commit:
Make the requested changes so this builds properly.

Also fix emphasis line numbers to match what was being emphasized before
the change.

gundalow commented:
Great, thanks for this

jcsmith commented:
No problem, I'll look at giving other parts of the docs the same kind of treatment now that I have my feet under me.

gundalow commented:
That would be ace. I'd say file at a time. feel free to `@gundalow` in them and I'll review & merge as soon as they look good to avoid them hanging around and getting merge conflicts.
